README.md

```diff
@@ -166,6 +166,24 @@ Unlike testing and web scraping libraries you're used to, Panther:
 * supports custom [Selenium server](https://www.seleniumhq.org) installations
 * supports remote browser testing services including [SauceLabs](https://saucelabs.com/) and [BrowserStack](https://www.browserstack.com/)
 
+### Using the `ServerListener` to Always Have a Running Web Server
+
+When you use the Panther client, the web server running in background will be started at runtime and stopped at test's
+teardown.
+
+If you want to save a few performances and launch the server at PHPUnit startup, you can add the `ServerListener` to
```

Comment:
`If you want to save a few performances ...` -> `If you want to improve performance ...`